packages/terra-tabs/src/common-tabs/CommonTabButton.module.scss

```diff
@@ -0,0 +1,58 @@
+@import '../clinical-lowlight-theme/WorkspaceButton.module';
+@import '../orion-fusion-theme/WorkspaceButton.module';
```

Comment:
I think we are missing this file in theme folder

Reply:
Added https://github.com/cerner/terra-framework/pull/1678/commits/8b40dac988c07e97c671313470b8975ddf258da8